tests/test_plotting.py

```diff
@@ -136,7 +136,7 @@ def test_plot_network(leogo_test_data: dto.EnergySystemData, leogo_expected_resu
     simulator.result_object = leogo_expected_result
     op.plot_network(simulator, timestep=1)
 
-    op.plot_network(simulator, timestep=1, filename="testplot.png")
+    op.plot_network(simulator, timestep=1)
```

Comment:
I added this line to test the writing of the image to file (which only works if graphviz is installed). Could skip thist, but then also delete the duplicate line below.

Reply:
We can write it as a temporary file. I will changes this :)